docs/api/applyMiddleware.md

```diff
@@ -87,131 +81,74 @@ function apologize(fromPerson, toPerson, error) {
     error
   }
 }
-
 function withdrawMoney(amount) {
   return {
     type: 'WITHDRAW',
     amount
   }
-}
-
-// Even without middleware, you can dispatch an action:
-store.dispatch(withdrawMoney(100))
-
-// But what do you do when you need to start an asynchronous action,
-// such as an API call, or a router transition?
-
-// Meet thunks.
-// A thunk is a function that returns a function.
-// This is a thunk.
-
+} // Even without middleware, you can dispatch an action:
+store.dispatch(withdrawMoney(100)) // But what do you do when you need to start an asynchronous action, // such as an API call, or a router transition? // Meet thunks. // A thunk is a function that returns a function. // This is a thunk.
```

Comment:
Woah, what happened here? I'd just leave it as-is and not remove the line breaks above.